src/test/java/org/jabref/logic/citationkeypattern/CitationKeyGeneratorTest.java

```diff
@@ -1094,4 +1094,20 @@ void generateKeyWithWithFirstNCharacters() {
 
         assertEquals("newt-2019", generateKey(entry, "[auth4:lower]-[year]"));
     }
+
+    @Test
+    void generateKeyCorrectKeyLengthWithTruncateModifierAndUnicode() {
+        BibEntry bibEntry = new BibEntry();
+        bibEntry.setField(StandardField.AUTHOR, "Gödel, Kurt");
+
+        assertEquals(2, generateKey(bibEntry, "[auth:truncate2").length());
```

Comment:
Is there a missing bracket at the end?

```suggestion
        assertEquals(2, generateKey(bibEntry, "[auth:truncate2]").length());
```

Reply:
There is indeed. Should the missing bracket generate an error or should I leave it alone?